documentation/modules/rql/pages/user-guide.adoc

```diff
@@ -114,59 +120,73 @@ option=limit(_start_,_count_),sort(-+attribute1,attribute2,attributeN)
 
 === Sorting
 
-Sorts the data based on the specified attribute in the order indicated by the prefix (+ for ascending, - for descending) The attributes names are separated by a comma.
+Sorts the data based on the specified attribute in the order indicated by the prefix (+ for ascending, - for descending). The attribute names are separated by a comma.
+
 With user-defined sorting, the client is responsible for preventing an arbitrary order of the retrieved results.
 
 [subs="quotes"]
 ----
 option=sort(-+attribute1,attribute2,attributeN)
 ----
+//TODO ? Why -+? Wouldn'T it suffice to only add + and the attribute names? Is it required to always note down - and +? Does the order matter?
 
 === Pagination
 
 Paging is controlled by the `option=limit(_start_,_count_)` part in the query string.
-Currently, the syntax is related to the offset pagination.
-Other strategy are currently not provided but may be added in the future e.g. cursor based pagination.
+
+Currently, the syntax is related to offset pagination.
+Other strategies are currently not provided but may be added in the future, e.g., cursor-based pagination.
+//TODO ? Users might want to know: What even is offset pagination?
 
 [subs="quotes"]
 ----
 options=limit(_start_,_count_)
 ----
+//TODO ? And I don't have to add numbers for start or count? I just literally add it like that to the query?
 
-== Attribute Names
+== Attribute names
 
-Attribute names used in `select` `filter` and `sort` statements are restricted to the following syntax: +
+Attribute names used in `select`, `filter`, and `sort` statements are restricted to the following syntax: +
 `[a-zA-Z_][a-zA-Z0-9/_]*`
+//TODO ? Isn't this rather indicating the allowed characters whereas the syntax comes in the examples below?
 
 *Examples:*
 
 name:: simple attribute named 'name'
+
 id:: simple attribute named 'name'
-location_v_2:: simple attribute named containing underscore and numbers
-_someStrangeThing:: attributes _may_ start with underscore but why would you?
+//TODO ? id ... named 'name'?
+
+location_v_2:: simple attribute name containing underscore and numbers
+
+_someStrangeThing:: attribute names _may_ start with underscore but why would you?
+//TODO ? Why not?
+
 address/street:: _nested_ attribute 'street' of parent attribute 'address'
 
-99NotValid:: *invalid* - attributes may not start with numbers
-/alsoNotValid:: *invalid* - attributes may not start with nesting separator
-willBeParsedButHasNoMeaning/:: *no meaning* - the parser will read this but it makes no sense and *may be removed in future releases*
+99NotValid:: *invalid* -- attributes must not start with numbers
+
+/alsoNotValid:: *invalid* -- attributes may not start with nesting separator
+
+willBeParsedButHasNoMeaning/:: *no meaning* -- the parser will read this but it makes no sense and *may be removed in future releases*
```

Comment:
Please remove it

Reply:
Done

https://github.com/bci-oss/semanticstack-rql/pull/69/commits/b3ccbfce0461cc81e2a560e8c7617827a07e2f21

https://github.com/bci-oss/semanticstack-rql/pull/69/commits/570101d4facef45d549da978e53e599913e0eda6